### ISSUE TITLE
Identity

### ISSUE THREAD

**Issue by seanmtracey** [ftlabs/cpp-extension#2]
Why do we request the [identity/identity.email](https://github.com/ftlabs/cpp-extension/blob/master/extension/manifest.json#L10) in our manifest? Is there functionality in the extension that's enabled only by having these permissions, or is it so we can find out more about who's using our extensions?

**Comment by AdaRoseCannon** [comment]
Yeah, I left it in there by default but I suppose we can remove it since this is aimed more externally than internally.
